src/client/pages/StaffViewAuthPage/index.js

```diff
@@ -0,0 +1,299 @@
+    false
+  );
+
+  const status = userData && userData.status;
+  const errorTransKey = new Map([
+    [
+      AUTH_STRINGS.statusCode.userNotFound,
+      "retrocert-login.invalid-user-error-staff-view",
+    ],
+    [
+      AUTH_STRINGS.statusCode.sessionTimedOut,
+      "retrocert-login.session-timed-out",
+    ],
+  ]).get(status);
+
+  const errorAlert = errorTransKey && (
+    <Row>
+      <div className="col-md-12">
+        <Alert variant="danger">{t(errorTransKey)}</Alert>
+      </div>
+    </Row>
+  );
+
+  const handleSubmit = (event) => {
+    const form = event.currentTarget;
+    const isValid = form.checkValidity();
+
+    event.preventDefault();
+    event.stopPropagation();
+
+    setValidated(true);
+
+    if (!isValid) {
+      setShowGenericValidationError(true);
+      autoScroll({
+        y: TOP.y,
+        x: TOP.x,
+        behavior: BEHAVIOR.smooth,
+      });
+      return;
+    }
+
+    setShowGenericValidationError(false);
+
+    const month = dobMonth.length < 2 ? "0" + dobMonth : dobMonth;
+    const day = dobDay.length < 2 ? "0" + dobDay : dobDay;
+
+    fetch(AUTH_STRINGS.apiPath.login, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        lastName: lastName.trim(),
+        dob: `${month}-${day}-${dobYear}`.trim(),
+        ssn: ssn.inputmask
+          ? ssn.inputmask.unmaskedvalue()
+          : ssn.trim().replace(/-/g, ""),
+      }),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setUserData(data);
+        if (data.authToken) {
+          // Session storage is destroyed when the tab is closed! That's a bit weird.
+          // If we want to allow the user to use multiple tabs, we could sync the
+          // value across tabs:
+          // https://medium.com/@marciomariani/sharing-sessionstorage-between-tabs-5b6f42c6348c
+          sessionStorage.setItem(AUTH_STRINGS.authToken, data.authToken);
+
+          if (data.confirmationNumber) {
+            // The user has already completed the retro-certs process.
+            history.push(routes.retroCertsConfirmation, { isReturning: true });
+          } else {
+            const completedWeeks = weeksCompleted(
+              data.formData,
+              data.programPlan
+            );
+            if (completedWeeks === 0) {
+              history.push(routes.retroCertsWeeksToCertify);
+            } else {
+              history.push(
+                `${routes.retroCertsCertify}/${fromIndexToPathString(
+                  data.weeksToCertify[
+                    Math.min(completedWeeks, data.weeksToCertify.length - 1)
+                  ]
+                )}`,
+                { returningUser: true }
+              );
+            }
+          }
+        }
+      })
+      .catch((error) => console.error(error));
+  };
+
+  const handleChange = (event, setter) => {
+    setter(event.target.value);
+  };
+
+  useEffect(() => {
```

Comment:
Let's confirm with @nguyen-tina, but I believe we don't want to hide the SSN. We can just use a regular text input.

Reply:
That's correct!

Reply:
great catch, thanks!